compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java

```diff
@@ -686,30 +751,41 @@ private BType setLiteralValueAndGetType(BLangLiteral literalExpr, BType expType)
         return literalType;
     }
 
+    private BType getFiniteTypeMatchWithFloatOrDecimalType(BType finiteType, Set<BType> memberTypes,
```

Comment:
```suggestion
    private BType getTypeMatchingFloatOrDecimal(BType finiteType, Set<BType> memberTypes,
```

shall we stick to this convention else where also